doc/classes/Environment.xml

```diff
@@ -98,6 +98,12 @@
 		<member name="fog_color" type="Color" setter="set_fog_color" getter="get_fog_color" default="Color( 0.5, 0.6, 0.7, 1 )">
 			The fog's [Color].
 		</member>
+		<member name="fog_exponential_enabled" type="bool" setter="set_fog_exponential_enabled" getter="is_fog_exponential_enabled" default="true">
+			If [code]true[/code], the exponential fog effect is enabled. When enabled, fog will appear depending on distance from the camera.
```

Comment:
```suggestion
			If [code]true[/code], depth fog will use exponential attenuation instead of using the curve defined in [member fog_depth_curve]. This is more physically accurate, but it gives less artistic control to the user.
			[b]Note:[/b] If exponential fog is enabled, [member fog_depth_begin] and [member fog_depth_end] are ignored. Instead, [member fog_exponential_density] controls how dense the depth fog is.
```